qiskit/transpiler/preset_passmanagers/common.py

```diff
@@ -195,7 +195,7 @@ def generate_unroll_3q(
     return unroll_3q
 
 
-def generate_embed_passmanager(coupling_map):
+def generate_embed_passmanager(coupling_map=None, target=None):
```

Comment:
Do we want to make this function just take a `coupling_map: Union[CouplingMap, Target]` as well (and update the docs)?

Reply:
I didn't think it was super critical here, I can change it if you'd prefer we'd basically just be reducing a single if statement in each of the pass manager construction functions.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/9263/commits/e0612d167acc168d8979a82a8dd562863a1da848